deploy/templates/azuredeploy.minimum.json

```diff
@@ -270,7 +270,7 @@
         "iothubTelemetryConsumerGroup": "telemetry",
         "iothubOnboardingConsumerGroup": "onboarding",
         "iothubEventsConsumerGroup": "events",
-        "iothubTunnelConsumerGroup": "tunnel",
+        "iothubTestConsumerGroup": "test",
```

Comment:
Why do we need `iothubTestConsumerGroup` ?

Reply:
We do not - but rather than creating more instability in the deployment I will handle this later when merging IAI and templates.